src/imports.rs

```diff
@@ -321,7 +327,7 @@ pub fn get_project_imports(
     ignore_type_checking_imports: bool,
 ) -> Result<ProjectImports> {
     let file_path = PathBuf::from(&file_path);
-    let absolute_source_root = PathBuf::from(&project_root).join(&source_root);
+    let absolute_source_root = PathBuf::from(&project_root).join(source_root);
```

Comment:
This is a surprising change, since I'd typically always use a reference when possible. Did a formatting tool suggest this?

Reply:
Yeah, this is all clippy